packages/jest-did-matcher/dist/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var _matchers = require('./matchers');
```

Comment:
dist files are generally not committed to source for packages.

This is related to the question regarding package publishing, however it can be handled later.

Reply:
I don't want to add dist but I prefer not to ask everybody who will run the test to add a large dependency of the dev dependency of jest-did-matcher nor run the build process. Is there any good way to approach this?

Honestly, I'm not used to package & ship complex packages in JavaScript. Especially packages involve build steps like Babel. Suggestions welcome.

Reply:
@shigeya formally in a lerna mono repo for packages with build steps...

`npm i` in the root will run `lerna bootstrap` as postinstall:

https://github.com/w3c/did-test-suite/blob/main/package.json#L22

This will automatically install all packages and then symlink them, so you don't need the `file://` prefixes.

the versions of dependencies must match to the link to be automatically created, otherwise it will try to pull from npm.

Reply:
@OR13 It works as you mentioned. Updated. Thanks.